src/App.vue

```diff
@@ -30,9 +30,10 @@
       }
     },
     computed: {
-      ...mapGetters(['current', 'tipSortBy']),
+      ...mapGetters(['settings', 'tipSortBy']),
     },
     methods: {
+      ...mapActions(['setLoggedInAccount', 'setTipsOrdering', 'updateTips', 'updateTopics', 'updateStats', 'updateCurrencyRates', 'setTipSortBy']),
```

Comment:
not sure about this. I personally like it better when its an explicit store communication.

Reply:
I think this is good practice :D